openapi/components/schemas/KycDocument/AddressMatches.yaml

```
@@ -1,56 +1,73 @@
 type: object
+description: Matched address data.
 properties:
   firstName:
-    description: Customer first name if it matches, null otherwise.
+    description: |-
+      First name of the customer.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: John
   lastName:
-    description: Customer last name if it matches, null otherwise.
+    description: |-
+      Last name of the customer.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: Doe
   line1:
-    description: Customer address if it matches, null otherwise.
+    description: |-
+      Address of the customer's residence.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: 36 Craven St
   city:
-    description: Customer city if it matches, null otherwise.
+    description: |-
+      Customer's city of residence.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: London
   region:
-    description: Customer region if it matches, null otherwise.
+    description: |-
+      Customer's region of residence.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: London
   postalCode:
-    description: Customer postal code if it matches, null otherwise.
+    description: |-
+      Postal code of the customer's residence.
+      This value is null if no match is found.
     type: string
     nullable: true
     example: WC2N 5NF
   wordCount:
-    description: Number of words in the document.
+    description: Total number of words in the document.
     type: integer
     example: 350
   uniqueWords:
-    description: Number of unique words in the document.
+    description: Total number of unique words in the document.
     type: integer
     example: 175
   date:
-    description: Date on the document proving the document is recent.
+    description: |-
+      Date detected on the document.
+      Use this field to determine if the document is recent.
     type: string
     format: date
     nullable: true
     example: 2021-01-01
   phone:
-    description: Phone of the company or agency that sent the document.
+    description: Phone number of the company or agency that issued the document.
     type: string
     nullable: true
     example: (123) 456-7890
   documentSubtype:
-    description: Interpreted subtype of the uploaded document.
+    description: |-
+      Interpreted subtype of the uploaded document.
+      For a list of available subtypes, see [KycDocumentSubtypes](https://github.com/Rebilly/api-definitions/blob/main/openapi/components/schemas/KycDocument/KycDocumentSubtypes.yaml).
     type: string
     nullable: true
     example: utility-bill
```

```suggestion
    description: Interpreted subtype of the uploaded document.
    type: string
    nullable: true
    allOf:
      - $ref: ./KycDocumentSubtypes.yaml
```

I don't it is good idea to place link to the GitHub repo, perhaps better to adjust schema to specify possible rerturned values.